docs/source/guide/ml.md

```diff
@@ -16,120 +16,80 @@ Integrate your model development pipeline with your data labeling workflow by ad
-- Dynamically pre-annotate data based on model inference results
-- Retrain or fine-tune a model based on recently annotated data
 
-For example, for an image classification task, the model pre-selects an image class for data annotators to verify. For audio transcriptions, the model displays a transcription that data annotators can modify. 
+At a high level, you'll need to do the following: 
 
-At a high level, do the following: 
 1. Set up an ML backend. You can either:
    - [Start an example ML backend](#Quickstart-with-an-example-ML-backend) or 
-   - [Create your own machine learning backend](ml_create.html) and then [Start your custom ML backend](#Start-your-custom-ML-backend-with-Label-Studio).
+   - [Create your own machine learning backend](ml_create)
 2. [Connect Label Studio to the ML backend](#Add-an-ML-backend-to-Label-Studio).
 
 ## Quickstart with an example ML backend
 
 Label Studio includes several example machine learning backends with popular machine learning models. See the [machine learning tutorials](ml_tutorials.html). Each example ML backend uses Docker Compose to start running the example ML backend server.
 
-To start an example machine learning backend with Docker Compose, do the following:
-1. Make sure port 9090 is available (or override it with --port parameter).
-2. Clone the Label Studio Machine Learning Backend git repository. From the command line, run the following:
+To start an example machine learning backend with Docker Compose, complete the following steps:
+
+1. Make sure port 9090 is available. For information changing ports, see [Modify the port](ml_create#Modify-the-port).
+2. Clone the Label Studio Machine Learning Backend git repository:  
   ```bash
-   git clone https://github.com/heartexlabs/label-studio-ml-backend  
+   git clone https://github.com/HumanSignal/label-studio-ml-backend.git 
    ```
-3. Change to the directory with the Docker Compose configuration file. From the command line, run the following:
+1. Change to the directory with the Docker Compose configuration file:
    ```bash
    cd label-studio-ml-backend/label_studio_ml/examples/the_simplest_ml_backend
    ```
-4. Start Docker Compose. From the command line, run the following:
+2. Start Docker Compose:
     ```bash
     docker-compose up
     ```
-The machine learning backend server starts becomes available at `http://localhost:9090`. You can now [add the machine learning backend to Label Studio](#Add-an-ML-backend-to-Label-Studio) and [set up your project](setup_project.html).
-
-## Start your custom ML backend with Label Studio
 
-After you [create your own machine learning backend](ml_create.html), you can start the ML backend server by following these instructions.
+You can access the machine learning backend server at `http://localhost:9090`. You can also use this URL to add the machine learning backend to Label Studio (see below).
```

Comment:
Talk about `host.docker.internal` if you're using Label Studio and the ML Backend inside of Docker

Reply:
Ok - added the note you authored.